plugins/inputs/consul/consul.go

```diff
@@ -4,6 +4,7 @@ import (
 	"net/http"
 
 	"github.com/hashicorp/consul/api"
+	consulStruct "github.com/hashicorp/consul/consul/structs"
```

Comment:
is this meant to be an exported type? I can't find any documentation that hashicorp meant for this to be used to extract metrics.